ver/us/splat.yaml

```diff
@@ -7595,73 +7595,62 @@ segments:
     vram: 0x80238000
     subsegments:
     - [0x6F10E0, c, goombario]
-    - [0x6F2290]
```

Comment:
These markers that explicitly say where the c code ends are still useful imo, even if they're not totally necessary